helm/ffc-demo-claim-service-infra/values.yaml

```diff
@@ -1,23 +1,45 @@
 namespace: ffc-demo
 
-subscriptionId: 
 serviceBusResourceGroupName: sndadpinfrg1401
 serviceBusNamespaceName: sndadpinfsb1401
 
 asoAnnotations:
   serviceoperator.azure.com/reconcile-policy: manage
 
 namespaceQueues:
-  - name: payment
+  - name: claim
+    owner: 'no'
+    roleAssignments:
+      - roleName: 'QueueReceiver'
   - name: calculation
+    roleAssignments:
+      - roleName: 'QueueSender'
 
 namespaceTopics:
   - name: schedule   
+    roleAssignments:
+      - roleName: 'TopicSender'
 
 postgresResourceGroupName: sndadpdbsrg1401
 postgresServerName: sndadpdbsps1401
 postgres:
   db:
     name: claim 
     charset: UTF8
-    collation: en_US.utf8
\ No newline at end of file
+    collation: en_US.utf8
+
+tags:
+  Environment: SND1
+  ServiceCode: ADP
+  ServiceName: ffc-demo-claim-service
+  ServiceType: Shared
+  Purpose: Define purpose
+  ManagedBy: ASO
+  kubernetes_cluster: SNDADPINFAK1401
+
+#Dummy placeholder values only for linting. NOT TO BE POPULATED
+subscriptionId: "dummySubscriptionId"
```

Comment:
these dummy values can be removed as per @ganeshg2491  coomment

Reply:
@DaRoszko working on a fix with Ganesh to remove all variables that doesn't concern developer. In progress.

Reply:
done